AnnoyancesFilter/sections/cookies_specific.txt

```diff
@@ -38,6 +38,7 @@ andoveradvertiser.co.uk,asianimage.co.uk,banburycake.co.uk,barryanddistrictnews.
 looper.com,fromdoctopdf.com,learnersdictionary.com,spanishdict.com,merriam-webster.com,digi24.ro,antena3.ro,wikihow.com.tr,wikihow.cz,wikihow.it,wikihow.jp,wikihow.vn,wikihow.com,mobil.se,digisport.ro,periscope-solutions.com###gdpr
 !
 portals.clio.me##.cookies-note-block
+rtl.hr###sdgdpr_modal_wrapper
```

Comment:
Cookie on safari? On Windows I cannot find this in code.

Reply:
Not only on Mac. Also on Windows.

<details><summary>Screenshot:</summary>

![image](https://user-images.githubusercontent.com/33488229/61896074-d476a080-af1c-11e9-847f-d566cdfbe785.png)
</details><br/>

Reply:
on my end `play.rtl.hr##.banner--gdpr` is required. Or `play.rtl.hr##.app__banner`